Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.hpp

```diff
@@ -176,16 +177,17 @@ class AdaptiveMultiVertexFinder {
         m_extractParameters([](T params) { return params; }),
         m_logger(std::move(logger)) {}
 
-  /// @brief Constructor for user-defined InputTrack_t type != BoundParameters
+  /// @brief Constructor for user-defined InputTrack_t type !=
+  /// BoundTrackParameters
   ///
   /// @param cfg Configuration object
-  /// @param func Function extracting BoundParameters from InputTrack_t object
+  /// @param func Function extracting BoundTrackParameters from InputTrack_t
+  /// object
   /// @param logger The logging instance
-  AdaptiveMultiVertexFinder(Config& cfg,
-                            std::function<BoundParameters(InputTrack_t)> func,
-                            std::unique_ptr<const Logger> logger =
-                                getDefaultLogger("AdaptiveMultiVertexFinder",
-                                                 Logging::INFO))
+  AdaptiveMultiVertexFinder(
+      Config& cfg, std::function<BoundTrackParameters(InputTrack_t)> func,
+      std::unique_ptr<const Logger> logger =
+          getDefaultLogger("AdaptiveMultiVertexFinder", Logging::INFO))
```

Comment:
just as an example that the current alignment also creates multi line changes by adding just a few chars

Reply:
Fair point.